backend/src/core/server.ts

```diff
@@ -40,14 +40,59 @@ export default class Server {
 
   private setupSockets(): void {
     const io = socketIo(this.httpServer);
-    io.on(Event.CONNECT, (socket: ExtendedSocket) => {
+    const videoChats: any = {};
+    io.on(Event.CONNECT, socket => {
       logger.debug(`Socket ${socket.id} connected.`);
-
+      logger.info(`Socket ${socket.id} connected.`);
       socket.on(Event.JOIN_ROOM, roomId => {
         new RoomSocketHandler(this.database, io, socket, roomId).initialize();
       });
 
-      socket.on(Event.DISCONNECT, (socket: ExtendedSocket) => {
+      socket.on("SEND_INVITE", invite => {
```

Comment:
use constants rather than hard strings

Reply:
Hahahahah, yeah i was being lazy here.

Reply:
we should probably make a separate enum to hold all video chat events